hazelcast/proxy/set.py

```diff
@@ -54,6 +63,7 @@ def add_all(self, items):
         return self._invoke(request, set_add_all_codec.decode_response)
 
     def add_listener(self, include_value=False, item_added_func=None, item_removed_func=None):
+        # type: (bool, event_collector(), event_collector()) -> Future[str]
```

Comment:
`event_collector` is not a type, `item_added_func` and `item_removed_func` are functions